### ISSUE TITLE
Don't work output S3 when "S3 - Default encryption" setting is disabled

### ISSUE THREAD

**Issue by motikan2010** [falcosecurity/falcosidekick#234]
**Describe the bug**

The below error has occurred when output AWS S3. 
 `panic: runtime error: invalid memory address or nil pointer dereference`

**How to reproduce it**

1. [Terminal A] Run falcosidekick on Docker.
```
# docker run -p 2801:2801 -e AWS_REGION="ap-northeast-1" -e AWS_S3_BUCKET="test-falcosidekick" falcosecurity/falcosidekick:2.22.0
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
2021/04/21 08:15:59 [INFO]  : Enabled Outputs : [AWSS3]
2021/04/21 08:15:59 [INFO]  : Falco Sidekick is up and listening on :2801
```

2. [Terminal B] Send test request.
```
# curl "http://localhost:2801/" -d'{"output":"16:31:56.746609046: Error File below a known binary directory opened for writing (user=root command=touch /bin/hack file=/bin/hack)","priority":"Error","rule":"Write below binary dir","time":"2019-05-17T15:31:56.746609046Z", "output_fields": {"evt.time":1507591916746609046,"fd.name":"/bin/hack","proc.cmdline":"touch /bin/hack","user.name":"root"}}'
```

3. [Terminal A] An error has occurred
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1964a5d]

goroutine 55 [running]:
github.com/falcosecurity/falcosidekick/outputs.(*Client).UploadS3(0xc0001e45a0, 0x0, 0x0, 0xc0009b86c0, 0x83, 0x5, 0xc0007fcb28, 0x16, 0x2c805996, 0xed470caec, ...)
    outputs/aws.go:164 +0x79d
created by main.forwardEvent
    handlers.go:185 +0xef8
```

**Expected behaviour**

- Terminal
<img width="800" src="https://user-images.githubusercontent.com/3177297/115580684-4b171c80-a302-11eb-98e1-bb2ec6924b0e.png">

- S3 setting
<img width="800" src="https://user-images.githubusercontent.com/3177297/115580823-6a15ae80-a302-11eb-9f41-bda82f96bed9.png">

**Environment**

- Falco version:
Checked by curl command.
- OS:
```
# cat /etc/os-release
NAME="CentOS Linux"
VERSION="8"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="8"
PLATFORM_ID="platform:el8"
PRETTY_NAME="CentOS Linux 8"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:8"
HOME_URL="https://centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"
CENTOS_MANTISBT_PROJECT="CentOS-8"
CENTOS_MANTISBT_PROJECT_VERSION="8"
```
- Kernel:
```
Linux ip-172-31-42-91.ap-northeast-1.compute.internal 4.18.0-240.1.1.el8_3.x86_64 #1 SMP Thu Nov 19 17:20:08 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```
- Installation method:
Docker (Podman)
```
# docker -v
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
podman version 2.2.1
```

**Comment by Issif** [comment]
This issue will be closed when I'll release the new version with #242 fix from @Kaizhe

**Comment by Issif** [comment]
The last release solves this issue 👍
